spyder/plugins/maininterpreter/confpage.py

```diff
@@ -158,9 +159,48 @@ def setup_page(self):
         umr_layout.addWidget(umr_namelist_btn)
         umr_group.setLayout(umr_layout)
 
+        # Path anaconda executor
+        conda_group = QGroupBox(_("Path Conda executor"))
+        conda_label = QLabel(
+            _(
+                "Select the conda executor path"
+            )
+        )
+        conda_label.setWordWrap(True)
+
+        conda_layout = QVBoxLayout()
+        conda_layout.addWidget(conda_label)
+
+        self.cus_conda_check = self.create_checkbox(
+            _("Use custom conda executor"), "customConda"
+        )
+
+        conda_layout.addWidget(self.cus_conda_check)
+        self.conda_path = self.create_browsefile(
+            _('Conda executor path'),
+            'conda_path',
+            filters='*.exe',
+            validate_callback=validate_conda,
+            validate_reason=_("The selected file is not a valid "
+                              "conda executable"),
+        )
+        self.conda_path.setStyleSheet("margin-left: 3px")
+        self.conda_path.textbox.setMinimumWidth(400)
+        conda_layout.addWidget(self.conda_path)
+        conda_group.setLayout(conda_layout)
+
+        self.conda_path.setEnabled(
+            self.get_option('customConda')
+        )
+        self.cus_conda_check.checkbox.toggled.connect(
+            self.conda_path.setEnabled)
```

Comment:
```suggestion
        custom_conda_check.checkbox.toggled.connect(conda_path.setEnabled)
```

Fix code style and don't use attributes.